### ISSUE TITLE
zabbixctl working with zabbix 5.x

### ISSUE THREAD

**Issue by AlexNinaber** [kovetskiy/zabbixctl#33]
I encounter the following error with zabbixctl after upgrading Zabbix to 5.x, any ideas?

[root@master02 zabbixctl]# ./zabbixctl -T -p -y -k
 :: Requesting information about statuses of triggers 
694635  2021-11-15 12:32:31  HIGH  PROBLEM  NACK  ws001.cluster  Slurm daemon on ws001.cluster is not UP

:: Proceed with acknowledge? [Y/n]: Y
 :: Acknowledging specified triggers 
can't acknowledge triggers [694635]
\u2514\u2500 zabbix returned error while working with api method event.acknowledge
   \u2514\u2500 Invalid params.
      \u2514\u2500 Incorrect arguments passed to function.

**Comment by kovetskiy** [comment]
Yeah, probably they have changed the signature of event.acknowledge function.
